src/notifications/broadcast_example.cpp

```diff
@@ -19,12 +19,12 @@
  *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
  *
  */
-#include "client.h"
+#include "broadcast.h"
 
 int main(int argc, char *argv[]) {
-  char *ip = const_cast<char *>("127.0.0.1");
-  Notify::NotifyClient client(ip, 8080);
-  std::string msg = "Hello World";
-  client.sendMessage(msg);
+  char *ip = const_cast<char *>("192.168.1.255");
+  Notify::NotifyBroadcast broadcaster(ip, 121121);
+  std::string msg = "Your back is broken :-( \n April Fools!";
```

Comment:
erm

Reply:
Just celebrating the day. Changed back now